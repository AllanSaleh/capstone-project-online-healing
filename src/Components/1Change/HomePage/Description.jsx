import React from 'react';
import Therapists from './Images/therapists.svg';

function Description() {
  return (
    <div className="text-left bg-yellow-primary h-section w-full px-sides m-">
      <div className="text-5xl w-7/12 font-normal pt-20">
        Professional, licensed, and vetted therapists that you can trust
      </div>
      <div className="pt-12 pb-16">
        <img src={Therapists} alt="therapists" />
      </div>
      <div className="text-lg w-8/12 pb-20">
        Tap into the world's largest network of licensed, accredited, and experienced therapists who
        can help you with a range of issues including depression, anxiety, relationships, trauma,
        grief, and more. with our therapists, you get the same professionalism and quality you would
        expect from an in-office therapist, but with the ability to communicate whenever and however
        you want.
      </div>
      <div>
        <button
          type="button"
          className=" pt-4 pb-4 pl-12 pr-12 text-subtitle rounded-md bg-blue-dark border-transparent hover:bg-white hover:text-blue-dark hover:border-blue-dark"
        >
          LEARN MORE
        </button>
      </div>
    </div>
  );
}

export default Description;
