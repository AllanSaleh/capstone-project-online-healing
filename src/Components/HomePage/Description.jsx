import React from 'react';
import { useTranslation } from 'react-i18next';
import { useHistory } from 'react-router-dom';

import Therapists from './Images/therapists.svg';

export default function Description({ user }) {
  const history = useHistory();
  const { t } = useTranslation();

  const GoToRoute = () => {
    if (user.login && user.complete) history.push('/Booking1');
    else if (!user.login) alert(t('HomePage.LoginAlert'));
    else alert(t('HomePage.CompleteAlert'));
  };

  return (
    <div className="text-left bg-yellow-primary h-section w-full px-sides m-">
      <div className="lg:text-5xl text-lg font-bold lg:w-7/12 lg:font-normal lg:pt-20 pt-20 uppercase">
        {t('HomePage.Therapist')}
      </div>
      <div className="pt-12 pb-16">
        <img src={Therapists} alt="therapists" />
      </div>
      <div className="lg:text-lg text-sm lg:w-8/12 lg:pb-20">{t('HomePage.TherapistInfo')}</div>
      <div>
        <button
          onClick={GoToRoute}
          type="button"
          className="uppercase mt-6 ml-4 py-2 px-6 lg:py-4 lg:px-6 lg:text-subtitle text-base rounded-md bg-blue-dark border hover:bg-white hover:text-blue-dark hover:border-blue-dark"
        >
          {t('HomePage.Book')}
        </button>
      </div>
    </div>
  );
}
