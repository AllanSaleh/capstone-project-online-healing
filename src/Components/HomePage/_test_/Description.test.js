import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Description from '../Description';

it(' Description renders correctly', () => {
  const tree = renderer.create(<Description />).toJSON();
  expect(tree).toMatchSnapshot();
});
