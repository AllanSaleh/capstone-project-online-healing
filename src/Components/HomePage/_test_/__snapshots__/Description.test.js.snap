// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Description renders correctly 1`] = `
<div
  className="text-left bg-yellow-primary h-section w-full px-sides m-"
>
  <div
    className="lg:text-5xl text-lg font-bold lg:w-7/12 lg:font-normal lg:pt-20 pt-20 uppercase"
  >
    Professional, licensed, and vetted therapists that you can trust
  </div>
  <div
    className="pt-12 pb-16"
  >
    <img
      alt="therapists"
      src="therapists.svg"
    />
  </div>
  <div
    className="lg:text-lg text-sm lg:w-8/12 lg:pb-20"
  >
    Tap into the world's largest network of licensed, accredited, and experienced therapists who can help you with a range of issues including depression, anxiety, relationships, trauma, grief, and more. with our therapists, you get the same professionalism and quality you would expect from an in-office therapist, but with the ability to communicate whenever and however you want.
  </div>
  <div>
    <button
      className="mt-6 ml-4 py-2 px-6 lg:py-4 lg:px-6 lg:text-subtitle text-base rounded-md bg-blue-dark border hover:bg-white hover:text-blue-dark hover:border-blue-dark"
      onClick={[Function]}
      type="button"
    >
      BOOK AN APPOINTMENT
    </button>
  </div>
</div>
`;
